dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/environmental/SfdrEnvironmental.kt

```diff
@@ -1,26 +1,26 @@
 package org.dataland.datalandbackend.model.sfdr.categories.environmental
 
-import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrBiodiversity
-import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEmissions
-import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnergyPerformance
-import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrGreenhouseGasEmissions
-import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrWaste
-import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrWater
+import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnvironmentalBiodiversity
+import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnvironmentalEmissions
+import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnvironmentalEnergyPerformance
+import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnvironmentalGreenhouseGasEmissions
+import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnvironmentalWaste
+import org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories.SfdrEnvironmentalWater
```

Comment:
The order of the imports is different here compared to `SfdrEnvironmental.kt` in the devtools repo.
Two questions:
1.) How come?
2.) I suggest to fix it so there are no "weird" diffs if the SFDR will be updated via the devtools in the future.

Reply:
The ktlint does the new ordering. And it seems that ktlint in the dev-tools does something different in the ordering as ktlint in main repo.

Reply:
Ah ok then let's just accept it for now if it has been done automatically.